package.json

```diff
@@ -12,8 +12,9 @@
   },
   "keywords": [
     "self learning clinic",
-    "tdd",
+    "TDD",
     "eslint"
```

Comment:
I noticed that I could not install from your package.json, and this is because you are missing a comma here. Try to pay more attention to these little details.

Reply:
also, your test command does not work. Check the `scripts` section of your package.json, you will find that there is no location such as `test/test.js`. You have no `test` directory. Fix that.